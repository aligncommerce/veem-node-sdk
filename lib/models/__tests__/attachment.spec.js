import Attachment from 'models/attachment'

describe('Attachment Model', () => {
  it('should match the snapshot', () => {
    expect(Attachment.schema).toMatchSnapshot()
  })

  it('should define a valid schema', () => {
    expect(Attachment.schema).to.be.validJsonSchema
  })
})
