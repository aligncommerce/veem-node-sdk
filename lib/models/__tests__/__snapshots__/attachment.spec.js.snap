// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attachment Model should match the snapshot 1`] = `
Object {
  "properties": Object {
    "name": Object {
      "type": "string",
    },
    "referenceId": Object {
      "type": "string",
    },
  },
  "title": "Attachment",
  "type": "object",
}
`;
